src/metabase/driver/sql_jdbc/connection.clj

```diff
@@ -103,6 +105,19 @@ For setting the maximum, see [MB_APPLICATION_DB_MAX_CONNECTION_POOL_SIZE](#mb_ap
                     (long (/ qp.pipeline/*query-timeout-ms* 1000))))
   :setter     :none)
 
+(setting/defsetting jdbc-data-warehouse-debug-unreturned-connection-stack-traces
+  "Tell c3p0 to log a stack trace for any connections killed due to exceeding the timeout specified in
+  [[jdbc-data-warehouse-unreturned-connection-timeout-seconds]].
+
+  Note: You also need to update the com.mchange log level to INFO or higher in the log4j configs in order to see the
+  stack traces in the logs."
+  :visibility :internal
+  :type       :boolean
+  :default    false
+  :export?    false
+  :setter     :none
+  :doc        false) ; This setting is documented in other-env-vars.md.
```

Comment:
Instead of adding to the other-env-vars docs, you can use the docstring + `:doc` key to document the setting. The docs script will pick up both the docstring and the `:doc` string value (if any) for defsettings. The other-env-vars file is primarily intended to document settings not defined as `defsettings`.

Reply:
Yeah this was a question I had in [this thread](https://github.com/metabase/metabase/pull/49456#discussion_r1830299742) above. Wasn't sure what the "right" place to doc it would be. Basically, I added it to `other-env-vars.md` because that is where `MB_JDBC_DATA_WAREHOUSE_UNRETURNED_CONNECTION_TIMEOUT_SECONDS` is documented, and these two vars are really closely related, so made sense to me to have the docs in the same place / produced in the way.